server/@types/shared/models/Cas2NewApplication.ts

```diff
@@ -0,0 +1,17 @@
+/* generated using openapi-typescript-codegen -- do no edit */
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+export type Cas2NewApplication = {
```

Comment:
[question] Probably a comment for the API PR the name feels a bit misleading. Would `Cas2Applicant` be more appropriate? There doesn't seem to be any data here that's connected to the application itself.

Reply:
ah yeah I could see it being a big confusing, but it's a bit of a technicality on the API - although these fields refer to the Person, they are being saved on the Application database record. The `Cas2NewApplication` refers to like the packet of data needed to create a new Application, and all these fields get saved against the Application Entity (so they each refer to a column on the database for an Application). I feel like calling it `Cas2Applicant` might be confusing as well, because we'd be POSTing an `Applicant` to make an `Application`...

Reply:
OK, I get what you mean. My thought was as we were previously sending the `crn` that adding these extra fields would be an extension of that and could be categorised as information about the applicant, but I understand how these values referring to columns in the application database may confuse things on the API side.

Happy to leave as is 👍🏾